Complete rewrite of organizations-index.html

thomasd8 opened the pull request:
Flesh out landing page for organizations/enterprise with more reasons for using Thunderbird and add descriptions and links for more resources, esp. enterprise.thunderbird.net.

Sancus commented on website/organizations/index.html:
```diff
@@ -22,14 +22,79 @@ <h3 class="header-section">
+          {{ _('Stability, efficiency, and security for your emails') }}
+        </h3>
+        <p class="font-lg tracking-wider leading-loose mb-10 mt-0 p-links-blue">
+          {{ _('Provide your team with a stable communications solution which allows them to focus on getting their work done. The <a href="/">regular Thunderbird release</a> follows the so-called <a href="{firefox_release_calendar}">Extended Support Release (ESR) schedule</a> of the Mozilla platform. So to your advantage it will remain stable and feature-stable on that major version for approxim ...
```
I would take out `so-called`. Makes this phrase seem dismissive and/or sarcastic.

This whole paragraph seems kind of repetitive and it's also wrong. We've done minor updates twice a month since 78, and we also have been releasing features into those minor updates...I actually think the old paragraph was better and didn't make any already-broken promises...

I'm not sure what this new paragraph is adding other than length?

thomasd8 replied:
Magnus and I both didn't like that the old paragraph was diverting attention to Firefox and per old wording started out with what we do *not* have.
To address your concerns, I've simplified this paragraph, removed the broken promises, and added back the sentence about beta.

Sancus replied:
Fair enough. I apologize if that was a bit strongly worded. I'm a fan of brevity where possible. I think the new paragraph works well!

Sancus commented on website/organizations/index.html:
```diff
@@ -22,14 +22,79 @@ <h3 class="header-section">
+            {{ _('Advanced Search and Filter tools') }}
+          </li>
+          <li>
+            {{ _('Inbuilt message encryption (OpenPGP, S/MIME)') }}
```
Strictly speaking, this should be `Built-in` for "en-US" english.

thomasd8 replied:
Perfect!

Sancus commented on website/organizations/index.html:
```diff
@@ -22,14 +22,79 @@ <h3 class="header-section">
+            {{ _('For more information about deploying and configuring Thunderbird in business and enterprise environments, please visit our enterprise deployment portal: <a href="{thunderbird_etn}">enterprise.thunderbird.net</a>.')|f(thunderbird_etn=url('thunderbird.enterprise.documentation')) }}
+          </li>
+          <li>
+            {{ _('You can also join the <a href="{tb_list_enterprise}">Thunderbird Enterprise mailing list on Topicbox</a> to connect with other administrators and share advice on Thunderbird deployment (Please note: The <a href="tb_oldlist_enterprise">old tb-enterprise mailing list</a> has been retired).')|f(tb_list_enterprise=url('thunderbird.lists.enterprise'), tb_oldlist_enterprise=url('thund ...
```
Do we really want to be linking to the old mailing lists? Is there important stuff people might need to search for in there? I feel like we shouldn't be mentioning outdated resources.

thomasd8 replied:
Afasik there IS important stuff in there, but maybe you're right, this isn't the right place to refer to collections of  wisdom from the past.

Sancus commented on website/organizations/index.html:
```diff
@@ -22,14 +22,79 @@ <h3 class="header-section">
+            {{ _('Implements accessibility standards') }}
+          </li>
+          <li>
+            {{ _('Open source code - transparency without proprietary mysteries') }}
```
I like this phrase.

thomasd8 replied:
Thanks 😃

Sancus reviewed (changes requested):
Looks good other than a few nitpicks and minor technical issues! Thanks for doing this.

thomasd8 commented on website/organizations/index.html:
```diff
@@ -22,14 +22,79 @@ <h3 class="header-section">
         </h3>
         <p class="font-lg tracking-wider leading-loose mb-6">
-          {{ _('Unlike the <a href="{firefox_esr}">Mozilla Firefox ESR</a>, Thunderbird does not provide a separate ESR release. This is because the <a href="/">regular Thunderbird release</a> is already an ESR release. It remains stable and on that major version for approximately one year. Only <a href="/#channel">Thunderbird Beta</a> follows the more frequent major version releases.')|f(firefox ...
+          {{ _('Thunderbird comes with a number of key benefits which make it an excellent choice for organizational deployments of any size. And it&#39;s more than just email: As an all-in-one communication platform with integrated calendar and chat, Thunderbird is all about efficiency. Whether you are a small business or organization, a government institution, or a large enterprise - Thunderbir ...
```
thanks, good catch

thomasd8 commented:
Thanks for the review! I've addressed all of the review comments and added new commits; let me know if that's the right thing to do wrt github workflows.

Sancus commented:
> 
> 
> Thanks for the review! I've addressed all of the review comments and added new commits; let me know if that's the right thing to do wrt github workflows.

Yeah, adding commits is good! Makes it easy to see changes and they all get squashed down to 1 by the final merge anyhow.

Sancus commented:
Thanks for this update!
